io.github.purplehorrorrus.Meridius.metainfo.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is something you should ideally maintain and distribute from your own repo/with the tarball.

Reply:
Does this mean I can delete these files from this repository?

Reply:
As long as you ship them with the tarball and make sure that either the manifest or the build system install these, yeah.

Reply:
Got it